index.html

```diff
@@ -256,53 +256,112 @@ <h1>Terminology</h1>
+	       Introductions, however, do not include any security or privacy controls and
+	       so should not provide metadata directly. 
+         Instead, the URLs provided by Introductions reference
+	       "Exploration" services.  Exploration services actually do provide metadata,
+         but only after suitable authentication and access controls have been applied.
+        </p>
+	      <p>
+	       The Discovery process can produce a <em>set</em> of URLs as output from
+	       its Introduction phase, even if only one Introduction mechanism is used
+         (some Introduction mechanisms can themselves return multiple URLs).
+	       The final output after the Exploration phase
+	       can also be a <em>set</em> of Thing Descriptions.
+        </p>
+        <p>
+	       Each URL provided by the Introduction phase always points at an 
+         Exploration service endpoint that will return a single Thing Description.
+	       In the simplest case this is the Thing Description of a Thing 
+	       describing an IoT endpoint device.  
+         In the case of self-describing Things an Introduction URL might
+	       even point directly at an endpoint provided by a 
+         Thing serving its own Thing Description.
+        </p>
+        <p>
+	       In general, however, Thing Descriptions might also be provided by other entities.
+         For example, 
+        </p>
+         <ul>
+         <li>brownfield devices not designed with the WoT 
+             specification in mind will not be capable of serving their
+             own Thing Descriptions; 
+         </li>
+         <li>battery-powered devices may not be online
+             most of the time; and 
+         </li>
+         <li>some devices may not be powerful enough to manage and 
+             serve TDs directly.  
+         </li>
+         </ul>
+        <p>
+         The Thing Description for such Things should be provided
+         by separate services.
+        </p>
+        <p>
+	       This document specifies two special cases that allow for more flexibility:
+	      </p>
+	      <ul>
+	        <li>A Thing Description referenced by an Introduction URL 
+            may describe a <a href="#exploration-td-type-thingdirectory">Thing Description Directory</a> service.  
+	          A Thing Description Directory service (which is still a Thing) 
+            maintains a (possibly dynamic) database of Thing Descriptions.
+            Thing Description Directories that maintain large numbers of 
+            Thing Descriptions may also support queries that can be used to 
+            selectively retrieve Thing Descriptions.
+	        </li>
+	        <li>
+            The second special case is a 
+            <a href="#exploration-td-type-thinglink">Thing Link</a>.  
+            A Thing Link is also a Thing Description,
+            but rather than describing a Thing directly holds a link to
+            a Thing Description hosted elsewhere.
```

Comment:
```suggestion
            A Thing Link is also a Thing Description,
            but rather than describing a Thing directly, it holds a link to
            a Thing Description hosted elsewhere.
```